ConnectionManager.java

```
@@ -0,0 +1,3 @@
+import java.sql.*;
+public class ConnectionManager{
+    public static Connection getConnection(){try{Connection con=DriverManager.getConnection("jdbc:mysql://localhost:3306/test","root","root");return con;}catch(Exception e){}return null;}}
```

Missing null check and proper exception handling. If the connection fails, the method should handle the exception appropriately rather than returning null silently.